<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notepadWindow</class>
 <widget class="QMainWindow" name="notepadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>notepadWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QWidget" name="settings" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="connectionBox">
         <property name="title">
          <string>Подключение</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>IP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ip">
              <property name="inputMask">
               <string/>
              </property>
              <property name="text">
               <string>127.0.0.1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Порт</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="port">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777210</height>
               </size>
              </property>
              <property name="maximum">
               <number>2000</number>
              </property>
              <property name="value">
               <number>1234</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Логин</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="login">
              <property name="text">
               <string>User1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="connectDisConnect">
            <property name="text">
             <string>Подключиться</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>connectDisConnect</zorder>
         <zorder></zorder>
         <zorder></zorder>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filesBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Файлы</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="files"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="createB">
              <property name="text">
               <string>Создать...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteB">
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="usersBox">
         <property name="title">
          <string>Полльзователи</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="users"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="textEditor" name="plainTextEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>textEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>texteditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
